tests/components/plex/test_media_search.py

```diff
@@ -57,6 +57,31 @@ async def test_media_lookups(
         )
     assert "Media for key 123 not found" in str(excinfo.value)
 
+    # Search with a different specified username
+    with (
+        patch(
+            "plexapi.library.LibrarySection.search",
+            __qualname__="search",
+        ) as search,
+        patch(
+            "plexapi.myplex.MyPlexAccount.user",
+            __qualname__="user",
+        ) as plex_account_user,
+    ):
+        plex_account_user.return_value.get_token.return_value = "token"
+        await hass.services.async_call(
+            MEDIA_PLAYER_DOMAIN,
+            SERVICE_PLAY_MEDIA,
+            {
+                ATTR_ENTITY_ID: media_player_id,
+                ATTR_MEDIA_CONTENT_TYPE: MediaType.EPISODE,
+                ATTR_MEDIA_CONTENT_ID: '{"library_name": "TV Shows", "show_name": "TV Show", "username": "Kids"}',
```

Comment:
So wait, this adds the username varaible to our media_content_id, but it won't be sent to the plex server, only to our side?

Reply:
That is correct. The username doesn’t go in this request, it is used to create the new server object for that user.